docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -213,6 +213,24 @@ corresponding week of the year.
 
 See the individual entry for the function.
 
+[[lob_functions]]
+== LOB Functions
+
+Trafodion provides following LOB functions to process LOB data.
+
+[cols="25%,75%"]
+|===
+| <<emptyblob_function,EMPTY_BLOB() Function>>   | Creates a dummy LOB handle with BLOB.
+| <<emptyclob_function,EMPTY_CLOB() Function>>   | Creates a dummy LOB handle with CLOB.
+| <<stringtolob_function,STRINGTOLOB Function>>  | Converts a simple string literal into LOB format.
+| <<filetolob_function,FILETOLOB Function>>      | Converts data from a local linux/hdfs file into LOB format.
+| <<buffertolob_function,BUFFERTOLOB_Function>>     | Takes an address and a size of an input buffer, and converts the data pointed to by that buffer into LOB format.
```

Comment:
Nit: To my ears, "Takes an address and the size of an input buffer..." sounds better. Probably the reason it sounds better to me is that I know the address is pointing to the input buffer. So, once we have said "an address" we have identified a specific input buffer, and "size" refers to the size of that specific buffer. So it requires the definite article rather than the indefinite article.

That said, perhaps another way of saying this is, "Takes the address and size of an input buffer, and converts the contents of that buffer into LOB format."

Reply:
OK, thanks Dave, for your detailed explanations. :)